stories/resizable-card-layout/resizable-card-layout.stories.js

```diff
@@ -0,0 +1,507 @@
+                            <div class="fd-card__title-area">
+                                <div class="fd-card__title">Quick Links</div>
+                                <span class="fd-object-status fd-card__counter">4 of 20</span>
+                            </div>
+                        </div>
+                    </a>
+                    <div class="fd-card__content" role="group" aria-label="Card Content">
+                        <ul class="fd-list fd-list--no-border" role="list">
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--cart"></i>
+                                <span class="fd-list__title">List item 1</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--wrench"></i>
+                                <span class="fd-list__title">List item 2</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--leads"></i>
+                                <span class="fd-list__title">List item 3</span>
+                            </li>
+                            <li role="listitem" tabindex="0" class="fd-list__item">
+                                <i role="presentation" class="fd-list__icon sap-icon--batch-payments"></i>
+                                <span class="fd-list__title">List item 4</span>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="fd-resizable-card-layout__wrapper__card__icon-wrapper">
+                    <div class="fd-resizable-card-layout__wrapper__card__icon-wrapper--icon fd-resizable-card-layout__wrapper__resize__cursor--both"></div>
+                </div>
+            </div>
+            <div class="fd-resizable-card-layout__wrapper__resize__handle--horizontal"></div>
+        </div>
+
+        <div id="four" class="fd-resizable-card-layout__wrapper" style="width:20rem; height:35rem;">
+            <div class="fd-resizable-card-layout__wrapper__resize__handle--vertical"></div>
+            <div class="fd-resizable-card-layout__wrapper__card">
+                <div class="fd-card fd-card--object" role="region" aria-label="Object Card Example 1">
+                    <a class="fd-card__header" tabindex="0">
+                        <span
+                                    class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--thumbnail fd-card__avatar"
+                                    style="background-image: url('/assets/images/avatars/1.svg')"
+                                    role="presentation"
+                                    aria-label="John Doe"></span>
```

Comment:
I think when you have `role="presentation"`, [aria attributes should not be present.](https://dequeuniversity.com/rules/axe/4.1/presentation-role-conflict?application=axeAPI)
you might get an a11y violation like this:
![image](https://user-images.githubusercontent.com/53509521/107486000-854d9a80-6baa-11eb-90ca-98b1c3aa7133.png)

Reply:
fixed.